site2/docs/admin-api-namespaces.md

```diff
@@ -417,7 +417,7 @@ admin.namespaces().getNamespaceMessageTTL(namespace)
 
 #### split bundle
 
-Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. If bundle gets heavy with multiple live topics in it then it creates load on that broker and in order to resolve this issue, admin can split bundle using this command.
+Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. If a bundle has multiple live heavily-used topics in it, then it creates load on that broker and in order to resolve this issue an admin can split the bundle using this command permitting the new bundles to be spread across the cluster.
```

Comment:
1. what does "If a bundle has multiple live heavily-used topics in it" and "create load" mean? maybe we can refine it to be clearer?
2. Could we refine the last sentence as "... an admin splits the bundle using this command,  and spreads the new bundles across the cluster."?

Reply:
Made some suggestions based on your feedback. Thanks!